### PR TITLE
feat(material/autocomplete): add panelClass default option

### PR DESCRIPTION
**Pull request by andrewseguin**
Necessary for implementing the Google Material version internally, which relies on being able to provide options to the autocomplete to automatically apply a Google Material class

### CODE REVIEW

**Review by crisbeto — COMMENTED** [review]
Do the panel classes have to be on the `mat-autocomplete-panel` specifically or can they be one level up on the `cdk-overlay-pane`? If it's the latter, then this could be simplified by passing the `panelClass` value when creating the overlay here: https://github.com/angular/components/blob/master/src/material/autocomplete/autocomplete-trigger.ts#L637

**Comment by andrewseguin** [comment]
Sure - that works too - as long as its a parent class then we're good. I changed it so the options are now injected into the trigger as well and we pass through the panel class

File: src/material/autocomplete/autocomplete.spec.ts [hunk 1]
```diff
@@ -2119,6 +2119,39 @@ describe('MatAutocomplete', () => {
     }));
   });
 
+  describe('with panel classes in the default options', () => {
```
Comment (crisbeto) [inline src/material/autocomplete/autocomplete.spec.ts:2122]: Can you copy these tests into the MDC autocomplete too?

**Review by crisbeto — APPROVED** [review]
One more nit and it's LGTM on my end.

File: src/material/autocomplete/autocomplete.spec.ts (hunk 1 above)
Comment (andrewseguin) [reply src/material/autocomplete/autocomplete.spec.ts:2122]: Oh yeah for sure, good call

**Pull request merged**